testdata/json-project/sub-class-project.json

```diff
@@ -0,0 +1,67 @@
+{
+  "prefixes": {
+    "foaf": "http://xmlns.com/foaf/0.1/",
+    "dcterms": "http://purl.org/dc/terms/",
+    "cidoc-crm": "http://www.cidoc-crm.org/cidoc-crm/"
+  },
+  "$schema": "../../src/dsp_tools/resources/schema/project.json",
+  "project": {
+    "shortcode": "001E",
+    "shortname": "sub-classing",
+    "longname": "Project with complex sub-classing",
+    "descriptions": {
+      "en": "Project with complex sub-classing"
+    },
+    "keywords": [
+      "test",
+      "testing"
+    ],
+    "ontologies": [
+      {
+        "name": "onto",
+        "label": "Test ontology",
+        "comment": "This is the ontology of the project",
+        "properties": [
+          {
+            "name": "hasUri",
+            "super": [
+              "hasValue",
+              "https://schema.org/url"
+            ],
+            "object": "UriValue",
+            "labels": {
+              "en": "URI"
+            },
+            "gui_element": "SimpleText"
+          }
+        ],
+        "resources": [
+          {
+            "name": "SuperClass",
+            "super": "Resource",
+            "labels": {
+              "en": "Super Class"
+            },
+            "cardinalities": [
+              {
+                "propname": ":hasUri",
+                "gui_order": 1,
+                "cardinality": "1"
+              }
+            ]
+          },
+          {
+            "name": "SubClass1",
+            "super": [
+              ":SuperClass",
+              "cidoc-crm:E70_Thing"
```

Comment:
If line 57 is removed then the upload works perfectly.
If the superclass `cidoc-crm:E70_Thing` is added to `SuperClass` then the upload works too. The only situation in which I could produce this bug, was when a subclass had a reference to an external ontology.